main.go

```diff
@@ -68,20 +69,35 @@ func (m *MigrationRunner) Run(ctx context.Context, conn *sql.Conn, lastRanId int
 				continue
 			}
 			if _, err := conn.ExecContext(ctx, fileScanner.Text()); err != nil {
-				return fmt.Errorf("failed to execute migration %s: %w", file, err)
+				return fmt.Errorf("failed to execute migration %s: %w", file, err), updatedID
 			}
 		}
 		log.Printf("Executed migration: %s\n", file)
+		updatedID++
 	}
 
-	return nil
+	return nil, updatedID
 }
 
 // RunAll runs migrations for every org in orgs and then returns the updated migration state.
 func (m *MigrationRunner) RunAll(ctx context.Context, state *migration.MigrationState) (*migration.MigrationState, error) {
 	for _, org := range state.Orgs {
 		// TODO: after each successful migration run, update the org.LastRanMigrationID
-		_ = org
+		dbName := "store_" + org.Name
+		db, err := connectDB(dbName)
+		if err != nil {
+			return nil, err
+		}
+		conn, err := db.Conn(ctx)
+		if err != nil {
+			return nil, err
+		}
+		err, newID := m.Run(ctx, conn, org.LastRanMigrationID)
```

Comment:
Just adding some cleanup items 
```suggestion
		db, err := connectDB(dbName)
		if err != nil {
			return nil, err
		}
		defer db.Close()
		
		conn, err := db.Conn(ctx)
		if err != nil {
			return nil, err
		}
		defer conn.Close()
		err, newID := m.Run(ctx, conn, org.LastRanMigrationID)
```